### DISCUSSION TITLE
How to install in Mac?

### DISCUSSION THREAD

**Discussion by brryyy** [royshil/obs-backgroundremoval#551] [General]
Can you help me install this on Mac?

Whenever I download and install this, I can't seem to find the installed files that I need to put in the OBS main folder.

A video or step by step process how to do it would be really helpful.

Thanks

**Comment by royshil** [comment]
Which plugin is this in regards to?
Are you running the installer?
Your OBS was installed regularly?

**Reply by brryyy** [reply]
the macOS background remover for OBS.

yes, I ran but I can't see the files it installed.

Yes, it was installed properly
